scripts/xjoin-config/log4j.properties

```diff
@@ -0,0 +1,8 @@
+log4j.appender.CONSOLE=org.apache.log4j.ConsoleAppender
+log4j.appender.CONSOLE.layout=org.apache.log4j.PatternLayout
+log4j.appender.CONSOLE.layout.ConversionPattern=%d{ISO8601} %p %X{connector.context}%m (%c) [%t]%n
+connect.root.logger.level=INFO
```

Comment:
Would like to know what log level we prefer in development environment? I expect it to be Debug? then can this be same for all the configs and/or services?

Reply:
This logging is specific to kafka connect. Kafka connect is not a direct dependency of ROS. Our docker-compose logs are already crowded, I think it is better to keep it as INFO.